### PR TITLE
Feature/clickable row

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/features/shared/components/molecules/Table/Table.tsx [hunk 1]
```diff
@@ -52,14 +68,19 @@ const createLoadingData = (numColumns: number, numRows: number = 5) =>
   [...Array(numRows)].map(_ => [...Array(numColumns)].map(_ => ""))
 
 const Table: React.FC<Props> = ({ columns, loading, numLoadingRows, hasFixedColumn, noValuesPlaceholder, className, ...restProps }) => {
-  const data = loading
-    ? createLoadingData(columns.length, numLoadingRows)
-    : restProps.data
+  const data = restProps.data
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/molecules/Table/Table.tsx:71]: `const { data } = restProps`

File: src/app/features/shared/components/molecules/Table/Table.tsx [hunk 2]
```diff
@@ -52,14 +68,19 @@ const createLoadingData = (numColumns: number, numRows: number = 5) =>
...
 
   const fixedColumnWidth = hasFixedColumn
     ? columns[columns.length - 1].minWidth
     : undefined
 
+const onClick = (href: string | undefined, e: any) => { 
+  if (href) {
+    e.stopPropagation()
+    href && navigate(href)
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/molecules/Table/Table.tsx:80]: There is a duplicate check for `href` here. (Inside the if statement)

File: src/app/features/shared/components/molecules/Table/Table.tsx [hunk 3]
```diff
@@ -52,14 +68,19 @@ const createLoadingData = (numColumns: number, numRows: number = 5) =>
...
-  const data = loading
-    ? createLoadingData(columns.length, numLoadingRows)
-    : restProps.data
+  const data = restProps.data
 
   const fixedColumnWidth = hasFixedColumn
     ? columns[columns.length - 1].minWidth
     : undefined
 
+const onClick = (href: string | undefined, e: any) => {
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/molecules/Table/Table.tsx:77]: I believe the type for `e` is: `React.MouseEvent<HTMLTrElement>`. https://fettblog.eu/typescript-react/events/

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/features/shared/components/molecules/Table/Table.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/features/shared/components/molecules/Table/Table.tsx]: Fixed it

File: src/app/features/shared/components/molecules/Table/Table.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/features/shared/components/molecules/Table/Table.tsx]: Fixed it

File: src/app/features/shared/components/molecules/Table/Table.tsx (hunk 3 above)
Comment (remcohoff) [reply src/app/features/shared/components/molecules/Table/Table.tsx]: I forgot to refactor that one. Changed it to `React.MouseEvent<HTMLTableRowElement>`

**Pull request merged**